tokio/Cargo.toml

```diff
@@ -27,75 +27,70 @@ keywords = ["io", "async", "non-blocking", "futures"]
 default = [
   "blocking",
   "fs",
-  "io",
-  "net-full",
+  "io-util",
+  "net",
   "process",
   "rt-full",
   "signal",
   "sync",
   "time",
 ]
 
-executor-core = []
-blocking = ["executor-core", "sync"]
-fs = ["blocking", "io-traits"]
-io-traits = ["bytes", "iovec"]
-io-util = ["io-traits", "pin-project", "memchr"]
-io = ["io-traits", "io-util"]
+blocking = ["rt-core"]
+dns = ["blocking"]
+fs = ["blocking"]
+io-driver = ["mio", "lazy_static", "sync"] # TODO: get rid of sync
+io-util = ["pin-project", "memchr"]
 macros = ["tokio-macros"]
-net-full = ["tcp", "udp", "uds"]
-net-driver = ["io-traits", "mio", "blocking", "lazy_static"]
-rt-current-thread = [
-  "executor-core",
-  "time",
-  "sync",
-  "net-driver",
+net = ["dns", "tcp", "udp", "uds"]
+process = [
+  "io-util", # TODO: Get rid of
+  "libc",
+  "mio-named-pipes",
+  "signal",
+  "winapi/consoleapi",
+  "winapi/minwindef",
+  "winapi/threadpoollegacyapiset",
+  "winapi/winerror",
 ]
+# Includes basic task execution capabilities
+rt-core = []
```

Comment:
`rt-core` instead of `rt-current-thread`. Note, this feature flag does not imply `time` or `net`. It only pulls in the current-thread executor.

Reply:
Hmm, do you think there should still be a `rt-current-thread` flag for `rt-core` + `time` + `net` w/o the threadpool?

Reply:
I don't think so. The current-thread executor is available w/ `rt-full`. If you want **only** current-thread + time / net, you can pick the flags by hand.